evalit/_base.py

```diff
@@ -109,7 +187,32 @@ def load_yaml(config_yaml: TYPE_PATH) -> Dict[str, str]:
 
     def _sanity_check_config(self, cfg: Dict[str, str]) -> bool:
         """
-        Check if we have sufficient keys in the config dict.
+        Check if we have sufficient keys in the config dict based on the
+        `AbstractAutomation._CFG_KEYS` structure for keys.
+
+        Args:
+            ```cfg```: ```Dict[str, str]```
+                The configuration dictionary for the transfer
+
+        Returns:
+            If everything is fine, it returns True else raise error
+
+        Example of dictionary structure:
+
+            .. code-block: python
+
+                {
+                    "source_token": "a****",
+                    "source_secret": "***s**rd",
+                    "source_s3_endpoint": "ht*p**/127.0**.*:80*0",
+                    "source_s3_bucket": "***",
+                    "source_s3_region": "u****s***",
+                    "dest_token": "*d*i*",
+                    "dest_secret": "p**s****",
+                    "dest_s3_endpoint": "htt*://127.0.*.1:*0**",
+                    "dest_s3_bucket": "**s*",
+                    "dest_s3_region": "u*-e*s***",
+                }
```

Comment:
Wouldn't it be easier to have the same format as _CFG_KEYS?

Reply:
Ah ya. But since it was in the original codebase (from before the time I wasn't involved), so I went with it. But I agree...